stdlib/public/SDK/Foundation/CharacterSet.swift

```diff
@@ -72,9 +72,7 @@ internal final class _SwiftNSCharacterSet : _SwiftNativeNSCharacterSet, _SwiftNa
  
  This type provides "copy-on-write" behavior, and is also bridged to the Objective-C `NSCharacterSet` class.
 */
-public struct CharacterSet : ReferenceConvertible, Equatable, Hashable, SetAlgebra, _MutablePairBoxing {
-    public typealias ReferenceType = NSCharacterSet
-    
+public struct CharacterSet : SetAlgebra, _MutablePairBoxing {
```

Comment:
I'd also like to leave `Equatable, Hashable` here, even if they are inferred from `ReferenceConvertible`, because `ReferenceConvertible` is really an implementation detail and we shouldn't make people inspect it to find out those key behaviors of `CharacterSet`.